rules-reviewed/hibernate/hibernate-catchall.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="hibernate-catchall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Please leave those as optional as suggested. In many cases, upgrading to the latest available JPA / Hibernate version is a very bad idea as it implies high efforts (code rewrite) and a lot of testing. For applications being at the end of their lifecycle, this is definitely not an option.

The "true" recommendation for applications using hibernate is however missing: keeping the used embedded ORM as-is and disabling JPA in EAP 6 / 7.